content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -12,18 +12,26 @@ private: true
 The features described on this page are in beta. Contact your Customer Success Manager to learn more about them.
 </div>
 
-This guide assumes that you have configured [Database Monitoring][1] and are using [APM][2].
+This guide assumes that you have configured [Database Monitoring][1] and are using [APM][2]. Connecting APM and DBM injects APM trace identifiers into DBM data collection, which allows for correlation of these two data sources. This enables product features showing database information in the APM product, and APM data in the DBM product.  
 
-## Before you begin
 
-Supported databases
-: postgres, mysql
+## Before you begin
 
-Supported Agent versions
-: 7.36.1+
+<dl>
```

Comment:
Do we really need the raw html in here? I know sometimes we don't have a choice but I have to ask because the mix of markdown and html is one of my least favorite things.

Reply:
yeah agree it's gross. The impact of markdown-only is that the code formatting for the env variables at the bottom of this section don't play well with the formatting for the Before You Begin section. The options were:
1. don't have a nice box around that section
2. put the env variables in their own code section outside the Before you begin section
3. use html << this has the best customer-facing result so... here we are

possibly a 4th option was "spend like a LOT of time getting better at markdown" but... i didn't want to do that.

Reply:
> 2. put the env variables in their own code section outside the Before you begin section

I actually think this one isn't that bad. That's kind of part of the recommended setup which they should be doing (if not already done).

Reply:
Moved down to the setup section above the tabs